corehq/apps/data_interfaces/tests/test_pillow.py

```diff
@@ -120,6 +123,21 @@ def test_pillow_skips_commtrack_cases(self, mock_run_rules):
 
         mock_run_rules.assert_not_called()
 
+    def test_pillow_processes_restored_forms(self):
```

Comment:
Yeah, I'm not sure how to make this test more readable. The bug I ran in to was that undo-ing a close action on a case was not being processed. However, from the code's perspective, there is no identifying information that this is a close -- it looks the same as any other action that publishes a case without an associated form ID. So, publishing a case without an associated form ID is what is being tested here. Do you think I should change the name of the test? Otherwise, it probably requires an integration test, because the specifics of handling the undo action are enclosed in a specific view

Reply:
Maybe add a comment in the test expaining those details?